api/representation.go

```diff
@@ -13,14 +12,9 @@ import (
 // this allows for the task request to be extended
 type taskRequest oapigen.TaskRequest
 
-type taskRequestConfig struct {
-	config.TaskConfig
-	variables map[string]string
-}
-
 // ToTaskRequestConfig converts a taskRequest object to a Config TaskConfig object. It takes as arguments a buffer period,
 // and a working directory which are required to finalize the task config.
-func (tr taskRequest) ToTaskRequestConfig(bp *config.BufferPeriodConfig, wd string) (taskRequestConfig, error) {
+func (tr taskRequest) ToTaskRequestConfig() (config.TaskConfig, error) {
```

Comment:
Not necessary for this PR since I know this is the rebase - I was wondering what your thoughts are on renaming `ToTaskRequestConfig` since  it's now returning a regular task config? Maybe something like `ToTaskConfig`?

Reply:
good catch, I'll make the update.